libephemeris/cli/__init__.py

```
@@ -823,15 +823,22 @@ def config() -> None:
     click.echo(_b("IERS Earth orientation data"))
     iers_auto = os.environ.get("LIBEPHEMERIS_IERS_AUTO_DOWNLOAD", "")
     iers_dt = os.environ.get("LIBEPHEMERIS_IERS_DELTA_T", "")
-    click.echo(f"  Location: {data_dir}/iers_cache/")
+    try:
+        from ..iers_data import get_iers_cache_info
+
+        iers_dir = str(get_iers_cache_info()["cache_dir"])
+    except Exception:
```

Ruff (0.16.3)

[warning] Do not catch blind exception: `Exception`

(BLE001)